### ISSUE TITLE
Installation on Windows

### ISSUE THREAD

**Issue by mihaifireball** [llinkz/directTDoA#5]
Do you think that it is possible to adapt the code in order to run on Windows?
The TDoA algorithm is running great, separately, under Windows, with Python 3.8 and Octave 4.4.0.
Thank you!

**Comment by llinkz** [comment]
I'll see, last time i've tested it, was not working on my Win7 64bits and octave 4.2.1 or 4.2.2 back in July 2018. stay tuned

**Comment by mihaifireball** [comment]
I am using Windows 10.
I have managed (I'm not a programmer :) ) to make some modifications to the main scripts (directTDoA.py and plot_iq.py) in order to make the program to start (there were some incompatibility related to linux/windows python commands). Now the program starts, the nodes appear on the map, the kiwirecorder is working, but i am not able to start the tdoa process from the interface (the "Start TDoA proc" button is not working.) Also there are some problems regarding the killing some sub-processes in Windows.
[Here](https://drive.google.com/open?id=1kGwPVs9ty3jI9dBmeCtSch2ZATDWG_b1) is my progress so far.
Thank you!

**Comment by llinkz** [comment]
Good news, had time to work on it and it seems to work OK on a laptop (Windows 7 pro 64 bits / GNU Octave 4.4.0 / python 2.7.18rc1 or python 3.8.2)

The modified .py files are stored on this website : http://81.93.247.141/~linkz/directTDoA/dev/v6.00/

Note: I wrote a Windows setup.bat, so maybe you can test everything from the beginning: 
1/ Getting all the stuff from git again, in a fresh new directory
2/ Download and run setup.bat to see if it's working fine on Windows 10 (also because there are some patches to apply on some original kiwirecorder & TDoA code files)
3/ And finally replacing all concerned .py files by the ones from the website

Feedback appreciated

**Comment by mihaifireball** [comment]
Thank you for your great work and support.
I've made everything from the beginning, and it is working on Windows 10, with python 3.8.2.
I've had some problems with "mkoctfile" process because of MinGW installed in C:\ and probably the variables on the PATH were messed up. After removing MinGW, everything was ok. 
Also I found some little bugs:
1/  if I press "Start recording", then "Stop recording" and then "Start TDoA proc" it is not working. It gives me this error:
`Traceback (most recent call last):
  File "C:\Program Files\Python38\lib\tkinter\__init__.py", line 1883, in __call__
    return self.func(*args)
  File "directTDoA.py", line 2305, in start_stop_tdoa
    os.kill(PROC2_PID, signal.SIGTERM)  # kills the kiwirecorder.py process
OSError: [WinError 87] The parameter is incorrect`

2/ The "Restart GUI" button is not working. The error is:
  `C:\Program: can't open file 'Files\Python38\python.exe': [Errno 2] No such file or directory`

I will keep you updated with the testing :).

**Comment by llinkz** [comment]
Thanks for the feedback, both bugs were fixed and the new directTDoA.py is on the website.

1st bug was due to an "os.kill" to a process PID that does not exist anymore, for some reasons it was not a problem on linux..
2nd bug is apparently a known problem with some setup where python executable path is containing spaces, I've changed the way to restart the GUI and it looks ok now. 

I've never noticed that bug because on all previous setups here, python never had a space in its absolute path, so thanks for the focus.

**Comment by mihaifireball** [comment]
Everything seems to be ok now. 
The Help window can be made scrollable? because right now you need a very high resolution monitor in order to read the entire help.

You may close this issue, and waiting for the v6.00 release! :) 

Off topic: What is the procedure to change the map (directTDoA_map_grayscale_with_sea.jpg)?
Thank you!

**Comment by llinkz** [comment]
The Help and About menus are no longer tk message boxes, a web browser is used instead, because the texts are now on the github wiki.

Regarding the available World maps, I decided for some time to no longer include them in the git.
I have stored them here: http://81.93.247.141/~linkz/directTDoA/maps/

Download them to /maps directory and they will be available from the "Browse map folder" command.

FYI: Daniel, the other contributor to this project has created a complete file archive for Windows users, it will include light versions of GNU Octave + python and all necessary files, already patched and compiled to make directTDoA work as easily as possible
